{"failedTests":[{"testName":"parallel/test-runner-watch-mode","timestamp":1727885135547,"status":"FAILED","jobName":"node-test-binary-windows-js-suites","config":"node-test-binary-windows-js-suites » 3,win2019-COMPILED_BY-vs2022 #30652","buildUrl":"https://ci.nodejs.org/job/node-test-binary-windows-js-suites/RUN_SUBSET=3,nodes=win2019-COMPILED_BY-vs2022/30652/","buildNumber":30652,"builtOn":"test-rackspace-win2019_vs2019-x64-4","nodeVersion":"23.0.0","commitHash":"d8a51cf8f8ca256ce92b650071b126345afd442b","callStack":[{"upstreamBuild":66348,"upstreamProject":"node-test-commit-windows-fanned","upstreamUrl":"job/node-test-commit-windows-fanned/"},{"upstreamBuild":74962,"upstreamProject":"node-test-commit","upstreamUrl":"job/node-test-commit/"},{"upstreamBuild":62894,"upstreamProject":"node-test-pull-request","upstreamUrl":"job/node-test-pull-request/"},{"prId":"55231"}],"tap":{"ok":false,"name":"parallel/test-runner-watch-mode","id":960,"buffered":false,"tapError":null,"skip":false,"todo":": Fix flaky test","previous":null,"plan":null,"diag":{"severity":"flaky","exitcode":1,"stack":"▶ test runner watch mode\n  ▶ isolation: none\n    ✔ should run tests repeatedly (1870.5041ms)\n    ✔ should run tests with dependency repeatedly (1829.5209ms)\n    ✔ should run tests with ESM dependency (1849.8074ms)\n    ✔ should support running tests without a file (1820.5948ms)\n    ✔ should support a watched test file rename (1745.6731ms)\n    ✔ should not throw when delete a watched test file (2326.4677ms)\n    ✖ should run new tests when a new file is created in the watched directory (1861.8105ms) # This test is failing when isolation is set to none and must be fixed\n      AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n      \n      '✔ new-test-file.test.js (3.1762ms)\\n' +\n        '✔ test has ran (0.498ms)\\n' +\n        'ℹ tests 2\\n' +\n        'ℹ suites 0\\n' +\n        'ℹ pass 2\\n' +\n        'ℹ fail 0\\n' +\n        'ℹ cancelled 0\\n' +\n        'ℹ skipped 0\\n' +\n        'ℹ todo 0\\n' +\n        'ℹ duration_ms 248.9591\\n'\n      \n          at testCreate (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:156:14)\n          at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n          at async testWatch (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:166:26)\n          at async TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:202:9)\n          at async Test.run (node:internal/test_runner/test:935:9)\n          at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n        generatedMessage: true,\n        code: 'ERR_ASSERTION',\n        actual: '✔ new-test-file.test.js (3.1762ms)\\n✔ test has ran (0.498ms)\\nℹ tests 2\\nℹ suites 0\\nℹ pass 2\\nℹ fail 0\\nℹ cancelled 0\\nℹ skipped 0\\nℹ todo 0\\nℹ duration_ms 248.9591\\n...',\n        expected: /tests 1/,\n        operator: 'match'\n      }\n\n  ✔ isolation: none (13313.4455ms)\n  ▶ isolation: process\n    ✔ should run tests repeatedly (1709.8228ms)\n    ✔ should run tests with dependency repeatedly (1708.1296ms)\n    ✔ should run tests with ESM dependency (1735.5104ms)\n    ✖ should support running tests without a file (4327.9922ms)\n      AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n      \n      '✖ test.js (1149.3632ms)\\n' +\n        \"  'test failed'\\n\" +\n        '\\n' +\n        '✖ test.js (916.5556ms)\\n' +\n        \"  'test failed'\\n\" +\n        '\\n' +\n        '✔ test has ran (2.1053ms)\\n' +\n        'ℹ tests 3\\n' +\n        'ℹ suites 0\\n' +\n        'ℹ pass 1\\n' +\n        'ℹ fail 2\\n' +\n        'ℹ cancelled 0\\n' +\n        'ℹ skipped 0\\n' +\n        'ℹ todo 0\\n' +\n        'ℹ duration_ms 218.5551\\n'\n      \n          at testUpdate (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:82:14)\n          at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n          at async testWatch (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:163:26)\n          at async TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:186:9)\n          at async Test.run (node:internal/test_runner/test:935:9)\n          at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n        generatedMessage: true,\n        code: 'ERR_ASSERTION',\n        actual: \"✖ test.js (1149.3632ms)\\n  'test failed'\\n\\n✖ test.js (916.5556ms)\\n  'test failed'\\n\\n✔ test has ran (2.1053ms)\\nℹ tests 3\\nℹ suites 0\\nℹ pass 1\\n...\",\n        expected: /tests 1/,\n        operator: 'match'\n      }\n\n    ✔ should support a watched test file rename (1868.8472ms)\n    ✔ should not throw when delete a watched test file (2363.8852ms)\n    ✔ should run new tests when a new file is created in the watched directory (1622.4076ms)\n  ✖ isolation: process (15342.3745ms)\n✖ test runner watch mode (28658.1051ms)\nℹ tests 14\nℹ suites 3\nℹ pass 12\nℹ fail 1\nℹ cancelled 0\nℹ skipped 0\nℹ todo 1\nℹ duration_ms 28672.0016\n\n✖ failing tests:\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:197:7\n✖ should run new tests when a new file is created in the watched directory (1861.8105ms) # This test is failing when isolation is set to none and must be fixed\n  AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n  \n  '✔ new-test-file.test.js (3.1762ms)\\n' +\n    '✔ test has ran (0.498ms)\\n' +\n    'ℹ tests 2\\n' +\n    'ℹ suites 0\\n' +\n    'ℹ pass 2\\n' +\n    'ℹ fail 0\\n' +\n    'ℹ cancelled 0\\n' +\n    'ℹ skipped 0\\n' +\n    'ℹ todo 0\\n' +\n    'ℹ duration_ms 248.9591\\n'\n  \n      at testCreate (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:156:14)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async testWatch (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:166:26)\n      at async TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:202:9)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: true,\n    code: 'ERR_ASSERTION',\n    actual: '✔ new-test-file.test.js (3.1762ms)\\n✔ test has ran (0.498ms)\\nℹ tests 2\\nℹ suites 0\\nℹ pass 2\\nℹ fail 0\\nℹ cancelled 0\\nℹ skipped 0\\nℹ todo 0\\nℹ duration_ms 248.9591\\n...',\n    expected: /tests 1/,\n    operator: 'match'\n  }\n\ntest at test\\parallel\\test-runner-watch-mode.mjs:185:7\n✖ should support running tests without a file (4327.9922ms)\n  AssertionError [ERR_ASSERTION]: The input did not match the regular expression /tests 1/. Input:\n  \n  '✖ test.js (1149.3632ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✖ test.js (916.5556ms)\\n' +\n    \"  'test failed'\\n\" +\n    '\\n' +\n    '✔ test has ran (2.1053ms)\\n' +\n    'ℹ tests 3\\n' +\n    'ℹ suites 0\\n' +\n    'ℹ pass 1\\n' +\n    'ℹ fail 2\\n' +\n    'ℹ cancelled 0\\n' +\n    'ℹ skipped 0\\n' +\n    'ℹ todo 0\\n' +\n    'ℹ duration_ms 218.5551\\n'\n  \n      at testUpdate (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:82:14)\n      at process.processTicksAndRejections (node:internal/process/task_queues:105:5)\n      at async testWatch (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:163:26)\n      at async TestContext.<anonymous> (file:///C:/workspace/node-test-binary-windows-js-suites/node/test/parallel/test-runner-watch-mode.mjs:186:9)\n      at async Test.run (node:internal/test_runner/test:935:9)\n      at async Suite.processPendingSubtests (node:internal/test_runner/test:633:7) {\n    generatedMessage: true,\n    code: 'ERR_ASSERTION',\n    actual: \"✖ test.js (1149.3632ms)\\n  'test failed'\\n\\n✖ test.js (916.5556ms)\\n  'test failed'\\n\\n✔ test has ran (2.1053ms)\\nℹ tests 3\\nℹ suites 0\\nℹ pass 1\\n...\",\n    expected: /tests 1/,\n    operator: 'match'\n  }"},"time":29107.379,"fullname":"parallel/test-runner-watch-mode","closingTestPoint":false}}]}